File: C Programming/Competitive Programming/Coding Challenges 18/program2.c
/*
2. Accept number of rows and number of columns from user and display below
pattern.
Input : iRow = 4 iCol = 4
Output : A B C D
         a b c d
         A B C D
         a b c d
*/
#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i=0;
    int j=0;
    
printf("Output is : \n");

    for ( i = 1; i <=iRow; i++)
    {
        if (i % 2 != 0)
        {
            for ( j = 1; j <= iCol; j++)
                {
                    printf("%c\t",'A'+(j-1));
                }
        }
        else
        {
            for ( j = 1; j <= iCol; j++)
                {
                    printf("%c\t",'a'+(j-1));
                }
        }
        printf("\n");
    }
}

int main()
{
    int iValue1=0,iValue2=0;

    printf("Enter Number of Rows: ");
    scanf("%d",&iValue1);

    printf("Enter Number of Column: ");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 7/program2.java
/*
2. Accept number of rows and number of columns from user and display below pattern.
Input : iRow = 4 iCol = 4
Output :* * * *
        * * *
        * *
        *
*/
import java.util.*;

class program2
{
    public static void Pattern(int iRow,int iCol) 
    {
        int i=iRow,j=0;

        do {
            j=1;
            do {
                if (j<=i) 
                    {
                         System.out.print("*\t");
                    } 
                else 
                    {
                        System.out.print(" ");
                    }
                 j++;
               } while (j <= iCol);
               i--;
               System.out.println();   
        } while (i>=1);

    }

    public static void main(String[] args) 
    {
        Scanner sobj=new Scanner(System.in);

        int iValue1=0,iValue2=0;

        System.out.println("Enter number of rows :");
        iValue1=sobj.nextInt();

        System.out.println("Enter number of columns :");
        iValue2=sobj.nextInt();

        Pattern(iValue1, iValue2);

        sobj.close();
    }
}